ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpStageSpec.scala

```diff
@@ -97,4 +99,47 @@ trait CommonFtpSourceSpec extends BaseSpec {
     }
   }
 
+  "FtpIOSink" should {
```

Comment:
I don't know what Akka members team will think, but seeing that it is so difficult to test this particular case, I would let it be. I'd stick to `jimfs` as it's been very rewarding up until now.